eth-node/bridge/geth/waku.go

```diff
@@ -63,6 +63,16 @@ func (w *gethWakuWrapper) AddStorePeer(address string) (peer.ID, error) {
 	return "", errors.New("not available in WakuV1")
 }
 
+// SubscribeToPubsubTopic function only added for compatibility with waku V2
+func (w *gethWakuWrapper) SubscribeToPubsubTopic(topic string, optPublicKey *ecdsa.PublicKey) error {
+	// not available in WakuV1
+	return nil
+}
```

Comment:
Do you think also return an error here? Just for consistency, as `StorePubsubTopicKey` returns an error.